src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -1915,16 +1912,13 @@ public void ParenthesizedExpression_04()
             UsingStatement(@"switch (e) { case (((x: 3))): ; }", TestOptions.RegularWithoutRecursivePatterns,
```

Comment:
>case (((x: 3))): [](start = 42, length = 16)

Could you add a test with pattern `((3))` too?

Reply:
See existing test `ParenthesizedExpression_01`.

---
In reply to: [234822088](https://github.com/dotnet/roslyn/pull/31056#discussion_r234822088) [](ancestors = 234822088)